xdebug_trace_computerized.c

```diff
@@ -226,6 +226,50 @@ void xdebug_trace_computerized_function_return_value(void *ctxt, function_stack_
 	xdfree(str.d);
 }
 
+void xdebug_trace_computerized_assignment(void *ctxt, function_stack_entry *fse, char *full_varname, zval *retval, char *op, char *filename, int lineno TSRMLS_DC)
+{
+	xdebug_trace_computerized_context *context = (xdebug_trace_computerized_context*) ctxt;
+	xdebug_str str = XDEBUG_STR_INITIALIZER;
+	unsigned int j = 0;
+	char *tmp_str;
+
+	xdebug_str_add(&str, xdebug_sprintf("%d\t", fse->level), 1);
+    /* no function_nr */
+	xdebug_str_add(&str, "\t", 0);
+
+	tmp_str = xdebug_show_fname(fse->function, 0, 0 TSRMLS_CC);
+
+	xdebug_str_add(&str, "2\t", 0);
+    /* skip time index, memory usage */
+	xdebug_str_add(&str, "\t\t", 0);
+	xdebug_str_add(&str, xdebug_sprintf("%s\t", tmp_str), 1);
+	xdebug_str_add(&str, xdebug_sprintf("%d\t", fse->user_defined == XDEBUG_EXTERNAL ? 1 : 0), 1);
```

Comment:
I don't think function name and user defined need to be shown again, as it can be inferred from the "0" record for this function number, and it would just add additional information. Just skip them with ``\t`` like time index and memory usage.